src/PhpRenderer.php

```diff
@@ -72,8 +72,7 @@ public function setLayout(string $layout): void
         if ($layout === '' || $layout === null) {
             $this->layout = null;
         } else {
-            $layoutPath = $this->templatePath . $layout;
-            if (!is_file($layoutPath)) {
+            if (! $this->hasTemplate($layout)) {
```

Comment:
Let's remove the extra space in the if condition. `if (!$this->hasTemplate($layout))`